crates/holochain/src/test_utils/conductor_setup.rs

```diff
@@ -0,0 +1,279 @@
+#![allow(missing_docs)]
```

Comment:
What's the deal with this whole file being added in this PR? Did someone remove it recently?